client/blocks/inline-help/inline-help-search-results.jsx

```diff
@@ -40,7 +40,7 @@ class InlineHelpSearchResults extends Component {
 		// copied from client/me/help/main for now
 		const helpfulResults = [
 			{
-				link: 'https://en.support.wordpress.com/com-vs-org/',
+				link: 'https://en.support.wordpress.com/business-plan/',
```

Comment:
Possible future enhancement - move this URL to the [`support.js` file](https://github.com/Automattic/wp-calypso/blob/481a77443e2c69d4a47c18b67c94a8f8a414e1fe/client/lib/url/support.js).